lib/vquic/ngtcp2.c

```diff
@@ -1814,7 +1814,13 @@ static CURLcode ng_flush_egress(struct Curl_easy *data,
   int rv;
   ssize_t sent;
   ngtcp2_ssize outlen;
+  /* Send larger UDP datagram if we can set DF bit. */
+#if defined(__linux__) && defined(IP_MTU_DISCOVER) &&                         \
+    defined(IPV6_MTU_DISCOVER)
+  uint8_t out[1452];
```

Comment:
And maybe, since the smaller value is 1200 (right?), maybe you could just use the larger value unconditionally there and save the code form a complicated #ifdef setup?

Reply:
Fixed in 1c66f5b99687e40a74cb739e712075ecb2da3dc1
NGTCP2_ name is now defined in ngtcp2 library header, and now always use larger buffer.